x-pack/elastic-agent/pkg/composable/providers/kubernetes/config.go

```diff
@@ -28,4 +28,5 @@ type Config struct {
 func (c *Config) InitDefaults() {
 	c.SyncPeriod = 10 * time.Minute
 	c.CleanupTimeout = 60 * time.Second
+	c.Scope = "cluster"
```

Comment:
Good catch! I think this should actually default to `node`?

Reply:
Currently when I run it, it's `cluster` because this was not set. If it's set to `node`, with the provider determine the node name automatically?

Reply:
Yes, autodetection should kick in when you run in cluster, that said, we pass the `node` setting in our manifests as autodetection can fail in some scenarios

Reply:
Okay. Updated it to `node`.